drivers/gpio/gpio_pcal64xxa.c

```diff
@@ -271,16 +284,16 @@ static int pcal64xxa_port_get_raw(const struct device *dev, gpio_port_value_t *v
 	return rc;
 }
 
-static int pcal64xxa_port_set_raw(const struct device *dev, uint16_t mask, uint16_t value,
-				  uint16_t toggle)
+static int pcal64xxa_port_set_raw(const struct device *dev, pcal64xxa_data_t mask,
+				  pcal64xxa_data_t value, pcal64xxa_data_t toggle)
 {
 	const struct pcal64xxa_drv_cfg *drv_cfg = dev->config;
 	struct pcal64xxa_drv_data *drv_data = dev->data;
 	int rc;
-	uint16_t output;
+	pcal64xxa_data_t output;
 
-	LOG_DBG("setting port with mask 0x%04X with value 0x%04X and toggle 0x%04X", mask, value,
-		toggle);
+	LOG_DBG("setting port with mask 0x" PRIpcal_data " with value 0x" PRIpcal_data
+		" and toggle 0x" PRIpcal_data, mask, value, toggle);
```

Comment:
```suggestion
		" and toggle 0x%" PRIpcal_data, mask, value, toggle);
```

Reply:
and done, you can tell I don't use these often :-) did the same on the gpio-kbd driver, will adjust that as well